src/rtCORSUtils.cpp

```diff
@@ -20,38 +20,50 @@
 
 #include "rtUrlUtils.h"
 
-#include <sstream>
+#include <ctype.h>
 
-const char* USE_ACCESS_CONTROL_CHECK_ENV_NAME = "USE_ACCESS_CONTROL_CHECK";
+namespace
+{
+  static const char* ENV_NAME_ENABLED = "SPARK_CORS_ENABLED";
+}
 
-rtError rtCORSUtilsCheckOrigin(const rtString& origin, const rtString& reqUrl, const rtString& rawHeaders, rtString* errorStr)
+rtError rtCORSUtilsCheckOrigin(const rtString& origin, const rtString& url, const rtString& headers)
 {
-  bool enableCheck = getenv(USE_ACCESS_CONTROL_CHECK_ENV_NAME) != NULL;
-  if (!enableCheck)
+  static bool enable = false;
+  static bool didCheck = false;
+  if (!didCheck)
   {
-    // not enabled
-    return RT_OK;
+    didCheck = true;
+    const char* s = getenv(ENV_NAME_ENABLED);
+    if (s != NULL)
+    {
+      rtString envVal(s);
+      enable = 0 == envVal.compare("true") || 0 == envVal.compare("1");
+    }
+    else
+    {
+      enable = false;
```

Comment:
Let's enable CORS by default.  The environment variable can be used to disable it.